fvm/environment/account_creator.go

```diff
@@ -16,6 +16,7 @@ const (
 	FungibleTokenAccountIndex = 2
 	FlowTokenAccountIndex     = 3
 	FlowFeesAccountIndex      = 4
+	EVMAccountIndex           = 5
```

Comment:
now that the EVM bootstrap step is optional, we need to factor getting to this value in the following PRs.